ceph/cephfs/Dockerfile

```diff
@@ -14,12 +14,21 @@
 
 FROM centos:7
 
-ENV CEPH_VERSION "luminous"
-RUN rpm -Uvh https://download.ceph.com/rpm-$CEPH_VERSION/el7/noarch/ceph-release-1-1.el7.noarch.rpm && \
+ENV CEPH_RELEASE "luminous"
+ENV CEPH_VERSION "12.2.5"
+RUN rpm -Uvh https://download.ceph.com/rpm-$CEPH_RELEASE/el7/noarch/ceph-release-1-1.el7.noarch.rpm && \
   yum install -y epel-release && \
-  yum install -y --nogpgcheck ceph-common python-cephfs && \
+  yum install -y --nogpgcheck librados2-$CEPH_VERSION libradosstriper1-$CEPH_VERSION libcephfs2-$CEPH_VERSION librbd1-$CEPH_VERSION librgw2-$CEPH_VERSION && \
```

Comment:
It is not necessary to use --nogpgcheck (I know the existing Dockerfile was using it, that doesn't make it okay).  It's better (although still not ideal) to install the GPG key, like Rook does:
https://github.com/rook/rook/blob/master/images/ceph/Dockerfile.base#L18